Integration/TrustedFormIntegration.php

```diff
@@ -58,154 +59,162 @@ public function doEnhancement(Lead $lead)
-                            ) {
-                                $lead->addUpdatedField('trusted_form_created_at', $data->created_at);
-                                $persist = true;
-                            }
+                                    // Set new value for trusted_form_created_at from created_at
+                                    if (
+                                        !empty($data->created_at)
+                                        && $data->created_at !== $lead->getFieldValue('trusted_form_created_at')
+                                    ) {
+                                        $lead->addUpdatedField('trusted_form_created_at', $data->created_at);
+                                        $persist = true;
+                                    }
 
-                            // Set new value for trusted_form_expires_at from expires_at
-                            if (
-                                !empty($data->expires_at)
-                                && $data->expires_at !== $lead->getFieldValue('trusted_form_expires_at')
-                            ) {
-                                $lead->addUpdatedField('trusted_form_expires_at', $data->expires_at);
-                                $persist = true;
-                            }
+                                    // Set new value for trusted_form_expires_at from expires_at
+                                    if (
+                                        !empty($data->expires_at)
+                                        && $data->expires_at !== $lead->getFieldValue('trusted_form_expires_at')
+                                    ) {
+                                        $lead->addUpdatedField('trusted_form_expires_at', $data->expires_at);
+                                        $persist = true;
+                                    }
 
-                            // Set new value for trusted_form_share_url from share_url
-                            if (
-                                !empty($data->share_url)
-                                && $data->share_url !== $lead->getFieldValue('trusted_form_share_url')
-                            ) {
-                                $lead->addUpdatedField('trusted_form_share_url', $data->share_url);
-                                $persist = true;
-                            }
+                                    // Set new value for trusted_form_share_url from share_url
+                                    if (
+                                        !empty($data->share_url)
+                                        && $data->share_url !== $lead->getFieldValue('trusted_form_share_url')
+                                    ) {
+                                        $lead->addUpdatedField('trusted_form_share_url', $data->share_url);
+                                        $persist = true;
+                                    }
```

Comment:
A lot of the empty and set checks are repetitive, consider tucking it away into a private function for readability and changing it to something like: 
```
$updatableFields = [
    self::CERT_URL_FIELD,
    'created_at',
   // ...
];
foreach($updatableFields as $field) {
    if(!empty($data->{$field} && $data->{$field} !== $lead->getFieldValue($field)) {
        $lead->addUpdatedField('trusted_form_share_url', $data->share_url);
        $persist = true;
    }
}
```